addons/mass_mailing/static/src/js/snippets.editor.js

```diff
@@ -185,7 +185,8 @@ export class MassMailingSnippetsMenu extends snippetsEditor.SnippetsMenu {
     _onFullscreenBtnClick(ev) {
         $("body").toggleClass("o_field_widgetTextHtml_fullscreen");
         const full = $("body").hasClass("o_field_widgetTextHtml_fullscreen");
-        this.options.wysiwyg.$iframe.parents().toggleClass("o_form_fullscreen_ancestor", full);
+        const parent = this.options.wysiwyg.$iframe[0].parentElement;
+        parent.classList.toggle("o_mass_mailing_iframe_ancestor_fullscreen", full);
```

Comment:
Mmh, is that actually different? If it's a cosmetic change to remove jQuery stuff, would not do it in stable.

Reply:
yes it is different, because before the code was adding that class to every ancestor until the beginning of times (it was quite ridiculous, the <body> element even got the class)